kafka/0.5/Dockerfile

```diff
@@ -35,11 +43,9 @@ RUN echo "$MD5HASH  /tmp/kafka.tgz" | md5sum -c - &&\
 COPY ./log4j.properties $KAFKA_HOME/config/log4j.properties
 
 #
-# Change ownership and switch user
+# Allow random UID to use Kafka
 #
-RUN chown -R kafka $KAFKA_HOME && \
-    chgrp -R kafka $KAFKA_HOME
-USER kafka
+RUN chmod -R g+w,o+w $KAFKA_HOME
```

Comment:
@jpechane I'm getting an error at this line:

    Step 13/19 : RUN chmod -R g+w,o+w $KAFKA_HOME
    chmod: changing permissions of '/kafka/config/log4j.properties': Operation not permitted
    The command '/bin/sh -c chmod -R g+w,o+w $KAFKA_HOME' returned a non-zero code: 1

Seems like that file still has the wrong owner or something?